src/options/options.css

```diff
@@ -20,74 +19,73 @@ body.mobile .mobile-incompatible {
   opacity: 0.4;
 }
 
-ul {
-  margin: 0px;
-  padding: 0px;
-}
-li {
-  list-style-type: none;
-  margin-top: 10px;
-  padding: 0px;
+form {
+  margin: 16px 32px;
 }
-/* in a fieldset e.g. we use a condensed version */
-.condensed-list {
-  margin-left: 0px;
+
+section {
+  margin-bottom: 32px;
 }
 
-fieldset {
-  /* same border style as separators above */
-  border: 1px solid;
-  border-radius: 4px;
-  border-color: var(--grey-30); /* Firefox: --in-content-box-border-color */
+h1 {
+  margin: 32px 0 16px 0;
+  padding: 16px 0 0 0;
 
-  /* a big width, but not too big */
-  display: inline-block;
-  width: auto;
-  min-width: 60vw;
+  /* title30 */
+  font-size: 22px;
+  font-weight: 300;
 
-  margin-bottom: 4px;
+  border-top: 1px solid rgba(12, 12, 13, 0.15);
 }
 
-/* when a setting has a label *before* it, add some margin */
-label + .setting {
-  margin-left: 8px;
+h1.first {
+  border-top: none;
 }
 
-.helper-text {
-  display: block;
-  color: var(--grey-50);
+h2{
+  /* title20 */
+  font-size: 17px;
+  font-weight: 600;
+  margin: 32px 0 16px 0;
+}
 
-  margin-top: 4px;
+ul {
+  margin: 0;
+  padding: 0;
 }
 
-/* some margin to align with checkbox */
-input[type=checkbox] ~ .helper-text,
-input[type=radio] ~ .helper-text {
-  /* 4px margin-left + 16px size + 3px margin-right + 5px space (#text) */
-  margin-left: 28px;
+li {
+  list-style-type: none;
+
+  padding: 2px 0;
+  margin: 0 0 4px 0;
+
+  display: flex;
+  align-items: center;
 }
 
-/* when a link is used in a helper text, add margin */
-.helper-text > a {
-  margin-left: 4px;
-  overflow-wrap: none;
+li * {
+  /* https://design.firefox.com/photon/visuals/grid.html#spacing */
+  margin: 0 8px 0 0;
 }
 
-/* special options values */
+li.indent {
+  margin-inline-start: 28px;
+}
 
-#popupIconColor {
+li.helper-text {
+  /* caption20 */
+  font-size: 13px;
```

Comment:
I have the feeling that this is a little too small, or at least smaller than in the Firefox settings page. Also in the screenshot on https://design.firefox.com/photon/patterns/helpers.html#necessary, it looks bigger. (maybe just like usual text?)